src/components/Game.jsx

```diff
@@ -42,15 +44,13 @@ function Game() {
             it is your turn
           </label>
           <button type="button" className="turnBtn" onClick={() => setPlayerTurn(!playerTurn)}>End Turn</button>
-          {heroList.map((hero) => (
-            <Card
-              key={hero.id}
-              avatar={hero}
-              playerTurn={playerTurn}
-              fighterId={heroFighterId}
-              setFighterId={setHeroFighterId}
-            />
-          ))}
+          <Modal
+            avatarList={heroList}
+            avatarFighterId={heroFighterId}
+            setAvatarFighterId={setHeroFighterId}
+            show={showHero}
+            setShow={setShowHero}
+          />
```

Comment:
you should have here:

    <Modal>
      {heroList.map((hero) => (
        <Card
          key={hero.id}
          avatar={hero}
          playerTurn={playerTurn}
          fighterId={heroFighterId}
          setFighterId={setHeroFighterId}
       />
      ))}
    </Modal>